src/app/components/article/AllArticles.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Articles from './Articles';
+import DescriptionIcon from '../../icons/description.svg';
+
+const AllArticles = ({ routeParams }) => (
+  <Articles
+    articleTypeReadOnly={false}
+    defaultFilters={{ article_type: null }}
```

Comment:
Why is it necessary to explicit pass `null` here?

Reply:
Not needed; mostly for clarity.

Reply:
Or maybe it's more confusing that way...

Reply:
Done!